### ISSUE TITLE
Keyboard navigation inside Image selector gallery doesn't work correctly when the dropdown is top-aligned

### ISSUE THREAD

**Issue by sgauruseu** [enonic/app-contentstudio#1211]
1. Open wizard for new Selector content (Features)  .
2. Scroll the page and go to `Allow all images` selector.
3. Type `spum` , the dropdown opening upwards.
4. Press Arrow Up key.

Expected; Image in dropdown options should be selected.
Actual : Image is selected when Arrow Down key has been pressed.

![image](https://user-images.githubusercontent.com/3728712/69157595-396e8b80-0af6-11ea-9b73-0faba4a8f70b.png)
